index.html

```diff
@@ -10,30 +10,38 @@
       @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700&family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,700;1,800&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500;1,700&display=swap');
       </style>
     <link rel="stylesheet" href="query.css" />
+    <script src="js/script.js" defer></script>
     <title>My Portfolio</title>
 </head>
 <body>
     <header class="header">
-        <a class="logo" href="#">Rubanza Mark</a>
-          <a class="menu" href="#"
-            ><img src="img/IconMenu.png" alt="burger button"
-          /></a>
-          <nav class="topMenu d-flex flex-center">
-            <ul class="d-flex">
-                <li>
-                    <a href="#">Portfolio</a>
-                </li>
-                <li>
-                    <a href="#">About</a>    
-                </li>
-                <li>
-                    <a href="#">Contact</a>                
-                </li>
-            </ul>
-            <a href="mailto:markrubanza10@gmail.com" class="mailIcon">
-                <img src="img/Icon - Mail.png" alt="mailIcon">
-            </a>
-      </header>
+      <a class="logo" href="#">Rubanza Mark</a>
+      <nav class="topMenu d-flex flex-center">
+        <ul class="d-flex">
+          <li>
+            <a href="#">Portfolio</a>
+          </li>
+          <li>
+            <a href="#">About</a>
+          </li>
+          <li>
+            <a href="#">Contact</a>
+          </li>
+        </ul>
+        <a href="mailto:markrubanza10@gmail.com" class="mailIcon">
+          <img src="img/Icon - Mail.png" alt="mailIcon">
+        </a>
+      </nav>
+      <div onclick="toggle(this)">
+        <div class="peakMobile">
+          <a href="#"><img class="imageHamburguer" src="img/IconMenu.png" alt="burger button" /></a>
+          <ul class="newMobile">
+            <li id="portfolio"><a href="#portfolio-section">Portfolio</a></li>
+            <li id="about"><a href="#about-section">About</a></li>
+            <li id="contact"><a href="#contact-section">Contact</a></li>
+          </ul>
```

Comment:
- [ ] Well done on adding the hamburger menu but kindly remove it from desktop version because hamburger menu is a mobile menu so it has to be only on mobile screens. Keep rocking 💪 

Here you have a screenshot of your desktop version:

![image](https://user-images.githubusercontent.com/74506933/157293746-7bf31cef-6022-4d4a-8463-16a47c755d6e.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/157293909-3fbb60de-ab23-45fd-9025-371265f0a8c8.png)